src/components/PeopleList.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+// import { usePeople } from '../store/PeopleContext';
+
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const selectedPerson = slug ? slug : '';
```

Comment:
The `selectedPerson` variable is assigned the value of `slug` directly, which is already a string. The ternary operation is unnecessary. You can simplify this by directly using `const selectedPerson = slug;`.